tools/build_defs/hash/sha256_test.sh

```diff
@@ -16,6 +16,7 @@
 
 set -e
 
+# The following code produces a 120MB file (30*2^22 bytes)
```

Comment:
Thanks, but please also add to the comment that the size is 120 MB merely because that seems to be big enough for the purpose of this test. If I was reading this code, I'd be wondering why exactly is the size 120 MB.